apps/hash-ai-worker-ts/src/activities/flow-activities/shared/propose-entities-from-facts/propose-entity-from-facts.ts

```diff
@@ -417,8 +552,19 @@ export const proposeEntityFromFacts = async (params: {
       });
     }
 
+    /**
+     * @todo: consider validating fact IDs specified in the input properties
+     */
```

Comment:
I haven't encountered an instance where the LLM doesn't provide correct `factIds` here, so far I haven't encountered an LLM hallucinating an invalid UUID. It's more likely that it might miss some facts used, which is harder to account for.